WordPress/Classes/Models/AccountSettings.swift

```diff
@@ -32,4 +32,12 @@ extension AccountSettings {
         webAddress = managed.webAddress
         language = managed.language
     }
+    
+    var emailForDisplay : String {
+        if let unwrappedPendingEmail = emailPendingAddress where unwrappedPendingEmail.isEmpty == false {
+            return unwrappedPendingEmail
+        }
+
+        return email
+    }
```

Comment:
I think this would do the same thing and I find it a bit clearer:

```swift
return emailPendingAddress?.nonEmptyString() ?? email
```

`nonEmptyString` basically turns an empty string into a `nil`